### ISSUE TITLE
Markdown support?

### ISSUE THREAD

**Issue by ebiyu** [binyamin/eleventy-plugin-backlinks#8]
Are you planning to support not only wikilinks but also simple markdown links?

I implemented [prototype of markdown support](https://github.com/ebiyuu1121/eleventy-plugin-backlinks/tree/markdown).
If you are willing to add markdown support, I will create pull-request and I want to ask something to decide specifications.

**Comment by rubenwardy** [comment]
The correct way to do this, IMO, is to use an HTML parser like this: https://blog.rubenwardy.com/2023/10/29/eleventy-backlinks/

**Comment by binyamin** [comment]
@ebiyu Sorry for the wait. I was under the impression that people aren't using normal markdown links to cross-link in a knowledge base. That's what obsidian does, as well as plenty of other tools. If you have found differently, please LMK.

**Comment by rubenwardy** [comment]
I use backlinks on a blog as a way to show related blog posts - so it's useful for more than just knowledge bases.

But using a regex like this plugin is the incorrect way to implement backlinks, you should use a HTML parser like my blog post shows above :)

**Comment by binyamin** [comment]
Yeah, that makes sense. I haven't used backlinks, or eleventy even, in about 5 years. So this is basically in maintenance mode.
